app/main.py

```diff
@@ -1 +1,36 @@
-# write your code here
+import random
+import matplotlib.pyplot as plt
+
+
+def flip_coin():
+    heads_odds = {0: 0, 1: 0, 2: 0, 3: 0, 4: 0, 5: 0,
+                  6: 0, 7: 0, 8: 0, 9: 0, 10: 0}
```

Comment:
And what if the range was 100? I think it's better to generate dict in required range.